DataSetExplorer/DataSetSerializer/ViewModel/ColumnHeuristicsModel.cs

```diff
@@ -12,8 +12,11 @@ internal ColumnHeuristicsModel()
         {
             _heuristics = new Dictionary<CodeSmell, List<string>>();
             //TODO: Load from DB or config.
```

Comment:
Since you've already introduced this information in the appsettings.Development.json, would it be better to load the information from there?

If I'm not mistaken, we will refactor both this and the appsettings with this story https://app.clickup.com/t/u9nd8j so you can leave this for now.